custom_components/google_home/number.py

```diff
@@ -57,9 +57,11 @@ async def async_setup_entry(
 class AlarmVolumeNumber(GoogleHomeBaseEntity, NumberEntity):
     """Google Home Alarm Volume Number entity."""
 
-    _attr_unit_of_measurement = PERCENTAGE
+    _attr_native_unit_of_measurement = PERCENTAGE
     _attr_entity_category = EntityCategory.CONFIG
-
+    _attr_native_min_value, _attr_native_max_value = (0, 100)
```

Comment:
I prefer it on separate lines.
```suggestion
    _attr_native_min_value, = 0
    _attr_native_max_value = 100
```